### PR TITLE
added label data field for text overflow

### PR DESCRIPTION
**Pull request by delarocamcas**
Don't merge this yet, I am using it to test the PR pipeline webhooks on the new jenkins

### CODE REVIEW

**Comment by delarocamcas** [comment]
Don't merge this yet, I am using it to test the PR pipeline webhooks on the new jenkins

File: picturae_importer.py [hunk 1]
```diff
@@ -307,7 +307,6 @@ def taxa_author_tnrs(self, taxon_name, barcode):
         self.parent_author = taxon_list[0]
 
 
-
     def create_agent_list(self, row):
```
Comment (delarocamcas) [inline picturae_importer.py:310]: changed logic for adding agent names to agent dict to strip leading and trailing spaces as well as make sure that None and nan are not turned into strings and missed by isna()

File: picturae_importer.py [hunk 2]
```diff
@@ -425,7 +437,7 @@ def populate_fields(self, row):
 
         self.tax_notes = row.cover_notes
 
-        self.label_text = row.label_text
+        self.label_data = row.label_data
```
Comment (delarocamcas) [inline picturae_importer.py:440]: label_data used for text overflow from other fields.

File: picturae_importer.py [hunk 3]
```diff
@@ -891,7 +903,7 @@ def create_collector(self):
             agent_id = agent_dict['agent_id']
 
             if agent_id != '' and pd.notna(agent_id):
-                agent_id = agent_dict['agent_id']
```
Comment (delarocamcas) [inline picturae_importer.py:894]: this line was redundant due to line 903

**Review by foozleface — APPROVED** [review]

**Pull request merged**